go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/filecoin-project/specs-actors/v3 v3.1.1
 	github.com/filecoin-project/specs-actors/v4 v4.0.1
 	github.com/filecoin-project/specs-actors/v5 v5.0.4
-	github.com/filecoin-project/specs-actors/v6 v6.0.0-20211001193936-c3afe7fa3c5c
+	github.com/filecoin-project/specs-actors/v6 v6.0.0-20211007184712-855f60561dfc
```

Comment:
Hash is right but this should use the rc tag instead of the hash: https://github.com/filecoin-project/specs-actors/releases/tag/v6-rc-2

Reply:
+1

Reply:
Chatted offline - actor didn’t tag properly. Will have the tag upon final release